request/xhr.js

```diff
@@ -34,7 +34,9 @@ define([
 	has.add('native-xhr2-blob', function(){
 		if(!has('native-response-type')){ return; }
 		var x = new XMLHttpRequest();
-		x.open('GET', '/', true);
+		// URL should be fully-qualified as XMLHTTPRequest within a Worker
+		// created from a Blob does not support relative URL paths
+		x.open('GET', 'https://dojotoolkit.org/', true);
```

Comment:
I assume this is just for the purposes of a feature test (to see if we can open a URL)? It's not clear from the comment that you've arbitrarily picked dojotoolkit.org, but I could see this breaking if an app is behind a particular firewall blocking dojotoolkit.org ?